tool_collections/fastx_toolkit/fastx_barcode_splitter/fastx_barcode_splitter.xml

```diff
@@ -1,10 +1,14 @@
-<tool id="cshl_fastx_barcode_splitter" version="1.0.0" name="Barcode Splitter">
+<tool id="cshl_fastx_barcode_splitter" version="@VERSION@" name="Barcode Splitter">
```

Comment:
@lparsons would you mind if I pull in your fastx_barcode_splitter_enhanced tool as a replacement? I think your version is much more useful.